assets/base.css

```diff
@@ -1428,6 +1428,10 @@ button.shopify-payment-button__button--unbranded {
   display: flex;
 }
```

Comment:
@LucasLacerdaUX let's do this update if it is not high LOE. if it is, we can park.

Reply:
This is how it looks on the product page and I believe it was added to avoid shadows being cut. 

<img width="382" alt="image" src="https://user-images.githubusercontent.com/37168033/158892803-6c2b20f3-393d-431d-855e-ccf5bbb97ff6.png">

We should create an issue to revisit this instead of changing it on quick-add only.